TriblerGUI/widgets/trustpage.py

```diff
@@ -85,11 +85,16 @@ def __init__(self):
         self.statistics = None
         self.blocks = None
         self.byte_scale = 1024 * 1024
+        self.timer = None
 
     def initialize_trust_page(self):
         vlayout = self.window().plot_widget.layout()
         self.trust_plot = TrustPlotMplCanvas(self.window().plot_widget, dpi=100)
         vlayout.addWidget(self.trust_plot)
+        
+        self.timer = QtCore.QTimer()
```

Comment:
It seems that the GUI tests are failing. Please use:

```
from PyQt5.QtCore import QTimer
```

and simply use the `QTimer` object 👍